Makefile

```diff
@@ -100,6 +154,22 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
+# Get the current opm binary. If there isn't any, we'll use the
+# GOBIN path
+opm:
+ifeq (, $(shell which opm))
+	@{ \
+	set -e ;\
+	opm_tool_url=https://api.github.com/repos/operator-framework/operator-registry/releases ;\
+	opm_tool_latest_version=$$(curl -s $$opm_tool_url | grep tag_name | grep -v -- '-rc' | head -1 | awk -F': ' '{print $$2}' | sed 's/,//' | xargs) ;\
+	curl -Lk https://github.com/operator-framework/operator-registry/releases/download/$$opm_tool_latest_version/linux-amd64-opm > $(GOBIN)/opm ;\
```

Comment:
$$opm_tool_latest_version 

two $ is fine here?

Reply:
yes, that's part of writing a multi-line shell script within a Makefile (as far as i know), otherwise every command is running in its own shell and we won't we able to use the variables that we set.